RELEASING.rst

```diff
@@ -10,4 +10,4 @@ Any other required changes related to the release should also be included in the
 
 When all changes are ready the branch should be tagged ``v<release-number>`` and merged.
 
-After released it is advisable to update https://github.com/ESSS/alfasim-sdk/blob/master/src/alfasim_sdk/_internal/version.py with the next release version (increase minor version, for example ``0.6.0`` to ``0.7.0``) and ``CHANGELOG.rst`` to add a section to next unreleased version.
+After released it is strongly advisable to update https://github.com/ESSS/alfasim-sdk/blob/master/src/alfasim_sdk/_internal/version.py with the next release version (increase minor version, for example ``0.6.0`` to ``0.7.0``, note that this may affect alfasim release) and ``CHANGELOG.rst`` to add a section to next unreleased version.
```

Comment:
If the version number must be bumped (if there are changes) before an alfasim final release is created because "the current version is listed in alfasim's about window" being the previous version plugin authors will mislead to use the wrong sdk version.
Having it bumped right after a release avoid that kind of problem (also alphas and rcs will list the correct sdk version).

Reply:
Went ahead and opened https://github.com/ESSS/alfasim-sdk/pull/283. :+1: